lib/lexer.mll

```diff
@@ -59,7 +71,7 @@ and cram_text section = parse
   | "<-- non-deterministic" ws* (("command"|"output") as choice) eol
```

Comment:
If the user type something else than "command" or "output", the label will be ignored.
Also, I think we should let the `Label` module decide what are the right value.
I suggest exposing a function in `Label` that interpret a label from the "parsed label" representation (eg. `Label.interpret "non-deterministic" (Some (Eq, choice))`).
This will help extending this syntax in the future.

Reply:
Good idea, done.